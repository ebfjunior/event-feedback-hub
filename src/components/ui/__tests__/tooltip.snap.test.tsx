import React from 'react';
import { render } from '@testing-library/react';
import { Tooltip, TooltipTrigger, TooltipContent } from '../tooltip';

describe('ui/Tooltip (snapshot)', () => {
	it('renders structure', () => {
		render(
			<Tooltip>
				<TooltipTrigger>Trigger</TooltipTrigger>
				<TooltipContent>Content</TooltipContent>
			</Tooltip>,
		);
		expect(document.body).toMatchSnapshot();
	});
});


