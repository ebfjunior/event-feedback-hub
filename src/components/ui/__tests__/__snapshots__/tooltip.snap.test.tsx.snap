// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ui/Tooltip (snapshot) > renders structure 1`] = `
<body>
  <div>
    <button
      data-slot="tooltip-trigger"
      data-state="closed"
    >
      Trigger
    </button>
  </div>
</body>
`;
